var/spack/repos/builtin/packages/gatk/package.py

```diff
@@ -35,6 +35,8 @@ class Gatk(Package):
     homepage = "https://software.broadinstitute.org/gatk/"
     url      = "https://github.com/broadinstitute/gatk/releases/download/4.0.4.0/gatk-4.0.4.0.zip"
 
+    version('4.0.8.1', sha256='6d47463dfd8c16ffae82fd29e4e73503e5b7cd0fcc6fea2ed50ee3760dd9acd9',
```

Comment:
technically you don't need the `sha256=` spack infers the type from the hash length

Reply:
I'm trying to get people to use them.  I would like to eventually make the package files more explicit/readable by having them specify the hash.